awx/api/views/__init__.py

```diff
@@ -2150,7 +2150,7 @@ class InventoryUpdateList(ListAPIView):
 class InventoryUpdateDetail(UnifiedJobDeletionMixin, RetrieveDestroyAPIView):
 
     model = InventoryUpdate
-    serializer_class = InventoryUpdateSerializer
+    serializer_class = InventoryUpdateDetailSerializer
```

Comment:
There are still several occurrences of `InventoryUpdateSerializer` in this file, and I suspect you would want to change those to `InventoryUpdateDetailSerializer`. This is for the UI. Right on launch, it would probably anticipate getting the detail fields.